packages/transformers/graphql/src/GraphQLTransformer.js

```diff
@@ -1,57 +1,32 @@
 // @flow
 import {Transformer} from '@parcel/plugin';
 
-const IMPORT_RE = /^# *import +['"](.*)['"] *;? *$/;
-const NEWLINE = /\r\n|[\n\r\u2028\u2029]/;
+import {parse, print, Source} from 'graphql/language';
+import {stripIgnoredCharacters} from 'graphql/utilities';
+import {processDocumentImports} from 'graphql-import-macro';
```

Comment:
You need to use `options.packageManager.require` (just like the old graphql transformer did) because these two dependencies are not a `package.json#dependency` (and `graphql` is quite big to install). This way it will be autoinstalled into the users project on the first usage.